frontend/src/locales/en.json

```diff
@@ -138,6 +138,16 @@
       },
       "printPreviewNuxt": {
         "openPreview": "Open preview in new window"
+      },
+      "printReact": {
```

Comment:
i would keep the same order as in de.json

Reply:
Fixed in https://github.com/ecamp/ecamp3/pull/3036/commits/abe5bd69b9ce833ec7f3432f27a82c851004e91e